apps/website/src/data/promo.ts

```diff
@@ -0,0 +1,20 @@
+interface Promo {
+  title: string;
+  subtitle: string;
+  content: string;
+  link: string;
+  external: boolean;
+  excluded: string[];
+}
+
+const promo: Promo | null = {
+  title: "Christmas Sweaters & 2024 Calendars",
+  subtitle: "Available from December 1st",
+  content:
+    "Grab yourself an embroidered Christmas sweater and a 2024 calendar featuring a selection of our ambassadors, shipping worldwide and in time for Christmas.",
```

Comment:
I'm not actually using this for now, but could be used to add a section to the homepage or similar down the road